src/lib.rs

```diff
@@ -71,6 +71,23 @@ bitflags! {
     }
 }
 
+impl RandomXFlag {
+    /// Returns the recommended flags to be used.
+    ///
+    /// Does not include:
+    /// * FLAG_LARGE_PAGES
+    /// * FLAG_FULL_MEM
+    /// * FLAG_SECURE
+    ///
+    /// The above flags need to be set manually, if required.
+    pub fn get_recommended_flags() -> RandomXFlag {
```

Comment:
Should this not rather be named default?

Reply:
Default is all flags not set (FLAG_DEFAULT which maps to RANDOMX_FLAG_DEFAULT).

Reply:
So while this fn will return valid flags, it won't necessarily return FLAG_DEFAULT.